# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: 42a5cf2bfccd
# Full Hash: 42a5cf2bfccdcc633e2ab80dcb7aa9f58bc87998
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-03-11 21:51:10
# Description:
#   Bug 1758127 - Don't use tight bounding rect if glyph transform fails. r=jnicol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140746
# ==============================================================================

diff -r da54c46e2e6d -r 42a5cf2bfccd gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Fri Mar 11 07:49:06 2022 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Fri Mar 11 08:06:39 2022 +0000
@@ -210,4 +210,4 @@
 load 1730695.html
 load 1745775.html
 load 1757002.html
-
+load 1758127-1.html